react-ts/basic/src/App.tsx

```diff
@@ -11,7 +11,7 @@ export const PLUGIN_SCRIPT_SRC =
 function Playground() {
   return (
     <div>
-      <PexipEngagePlugin version="1.0.0" />
```

Comment:
In the docs we mention that we recommend to always add a version
https://github.com/skedify/frontend-mono/pull/2730/files#diff-738ab28141ad7e8362963ad5d3e43a30c13415dc11ef6152446e03733b17a86fR71

Or has this been moved somewhere else?